lib/model.js

```diff
@@ -2453,6 +2453,7 @@ Model.prototype.update = function(values, options) {
         }
       });
     }
+    return null;
```

Comment:
http://bluebirdjs.com/docs/warning-explanations.html#warning-a-promise-was-created-in-a-handler-but-none-were-returned-from-it